test/snippet/io/sequence_file/sequence_file_input_aminoacid.cpp

```diff
@@ -2,11 +2,11 @@
 
 #include <seqan3/io/sequence_file/input.hpp>
 
-auto input = R"(> TEST1
+auto input = R"(>TEST1
 ACGT
-> Test2
+>Test2
 AGGCTGA
-> Test3
+>Test3
```

Comment:
Are the "input" snippets really necessary to change? We accept also files that have whitespaces after `>`, so I guess it is okay to have an example that shows that.

Reply:
I guess to be consistent and show that these are now default "files" it is also okay. You already did the work, so let's keep it.